report-hazard/report-hazard.js

```diff
@@ -0,0 +1,96 @@
+const allPages = document.querySelectorAll('div.page');
+allPages[0].style.display = 'block';
+
+function navigateToPage(event) {
+    const pageId = location.hash ? location.hash : '#step1';
+    for (let page of allPages) {
+        if (pageId === '#' + page.id) {
+            page.style.display = 'block';
+        } else {
+            page.style.display = 'none';
+        }
+    }
+    return;
+}
+navigateToPage();
+
+//init handler for hash navigation
+window.addEventListener('hashchange', navigateToPage);
+
+
+class Report {
+    constructor(location, category, type, details, picture) {
+        this.location = location;
+        this.category = category;
+        this.type = type;
+        this.details = details;
+        this.picture = picture;
+    }
+}
+
+const reportArray = []
```

Comment:
we don't need the array, just console log the object and add a //TODO: Submit hazard report

Reply:
commit c991d5205f38a526b6499863a0b1c7ccdfa122ae